test_utils.py

```diff
@@ -0,0 +1,53 @@
+import utils
+import fetch
+
+def test_submission_endpts():
```

Comment:
Ideally, these tests would be parametrized (https://docs.pytest.org/en/latest/how-to/parametrize.html), however these tests should suffice for now.

Reply:
(same goes for the other tests in this file)